spec/integration/guest_view_spec.rb

```diff
@@ -3,11 +3,9 @@
 describe "the guest view", type: :feature do
   describe "the home page" do
 
-    before(:each) do
```

Comment:
I personally like the before(:each) in this case because visiting the root_path is required for all tests in that describe block